src/index.handlebars

```diff
@@ -268,12 +255,36 @@
                 <div class="row">
                     <div class="col-xs-12">
                         <div class="panel panel-ghost panel-btn">
-                            <a href="https://github.com/akeneo/pim-api-docs/blob/master/content/swagger/akeneo-web-api.json" target="_blank">
+                            <div class="panel-body">
```

Comment:
I think you can leave the link on the entire `div` as it was before. I did the same for the OpenAPI spec, and several people reported back to me that they thought the link wasn't working.